### PR TITLE
fix: support wallet api accountId manifests

### PR DESCRIPTION
**Pull request by sarneijim**
### 📝 Description

When manifest are using wallet-api account context is incorrect.

The fix have been using the new paraswap manifest version
https://ledgerhq.atlassian.net/browse/LIVE-7658?focusedCommentId=323265

### ❓ Context

- **Impacted projects**: LLD
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-7658

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Review by beths-ledger — APPROVED** [review]

**Comment by adrienlacombe-ledger** [comment]
do not merge, I would like to better understand what you are trying to achieve first

**Comment by sarneijim** [comment]
Pending to discuss with @adrienlacombe-ledger

**Review by IAmMorrow — CHANGES_REQUESTED** [review]
This component is currently responsible for blindly forwarding the parameters to the WebPlatformPlayer (and so to the live apps). This PR changes its behavior by making it react to some very specific param key for the Paraswap / 1Inch scenario. I think this is not optimal as it would effectively make the component smarter thank it should be and increase complexity / maintenance cost.

I think I would implement this logic where the redirect is initially done (probably in the swap logic ?). If we can't dynamically adapt to the manifest version, a feature flag is probably okay to pilot this since this is a one shot migration and not something that will keep being used in the future 🤞

File: apps/ledger-live-desktop/src/renderer/screens/platform/App.tsx [hunk 1]
```diff
@@ -53,6 +58,13 @@ export default function PlatformApp({ match, appId: propsAppId, location }: Prop
   const localManifest = useLocalLiveAppManifest(appId);
   const remoteManifest = useRemoteLiveAppManifest(appId);
   let manifest = localManifest || remoteManifest;
+  // Convert accountId value when manifest is using wallet-api api.
+  if (manifest.apiVersion === "^2.0.0 || ^1.0.0") {
```
[the remark this answers is not in the document]
Comment (IAmMorrow) [reply apps/ledger-live-desktop/src/renderer/screens/platform/App.tsx:62]: FYI we have a semver package that should be used for any semver logic :)

**Pull request closed without merging**